### ISSUE TITLE
remote.NewImage() with remote.FromBaseImage() fails on Windows when provided a manifest list

### ISSUE THREAD

**Issue by natalieparellano** [buildpacks/imgutil#54]
E.g., 

`remote.NewImage(tags[0], authn.DefaultKeychain, remote.FromBaseImage("mcr.microsoft.com/windows/nanoserver:1809")` (a manifest list)

fails with 

`connect to repo store 'mcr.microsoft.com/windows/nanoserver:1809': no child with platform amd64/linux in index mcr.microsoft.com/windows/nanoserver:1809`

Whereas

`remote.NewImage(tags[0], authn.DefaultKeychain, remote.FromBaseImage("mcr.microsoft.com/windows/nanoserver:1809-amd64")` (an image index)

works just fine.

**Comment by ekcasey** [comment]
Makes sense. I think remote.NewImage, needs a `WithPlatform` option that we can pass through to `ggcr` so we can select the correct manifest from the list.

**Comment by jjbustamante** [comment]
Hi @ekcasey @natalieparellano, I am planning to give it a shot to this issue, right now I am receiving help to create a VM with windows where I can actually have administration permissions to install tools on it, once that's done, I will prepare my environment and try to reproduce the issue.
